epoxy-processor/src/main/java/com/airbnb/epoxy/processor/Memoizer.kt

```diff
@@ -359,6 +359,17 @@ class Memoizer(
             }
         }
     }
+
+    private val implementsModelCollectorMap = mutableMapOf<GeneratedModelInfo, Boolean>()
```

Comment:
to actually get good cache hits on this the parameter of the function should be `classElement: TypeElement`
then the map can be `mutableMapOf<Name, Boolean>()`, with `getOrPut(classElement.qualifiedName)` as the key to make sure it is consistent across type instances